frontend/src/test/java/com/google/tripmeout/frontend/service/PlaceServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package com.google.tripmeout.frontend.service;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth8.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.maps.GeoApiContext;
+import com.google.maps.errors.ApiException;
+import com.google.maps.errors.InvalidRequestException;
+import com.google.tripmeout.frontend.PlaceVisitModel;
+import com.google.tripmeout.frontend.service.PlaceService;
+import com.google.tripmeout.frontend.service.PlacesApiPlaceService;
+import java.io.IOException;
+import java.lang.InterruptedException;
+import java.util.List;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+//@RunWith(JUnit4.class)
+public class PlaceServiceTest {
+  Injector injector = Guice.createInjector(new PlaceServiceModule());
```

Comment:
Please initialize all these fields in a `@Before`-annotated setup method and do not retain the injector itself. You also shouldn't need an explicit `GeoApiContext` either.

 For example:

```java
@Inject
private PlaceService placeService;

@Before
public void setUp() {
  Injector injector = Guice.createInjector(new PlaceServiceModule());
  injector.injectMembers(this);
}
```